packages/rancher-logging/overlay/templates/loggings/aks/logging.yaml

```diff
@@ -15,6 +15,9 @@ spec:
     inputTail:
```

Comment:
One general downside here is that PSP's are likely going away soon, so this will likely need editing in the future.

Reply:
True, when we know what the PSP replacement will be, this will need some work again. But I think, as long as we offer PSP's in the UI when creating/configuring a cluster and have it in our hardening guide, our helm charts also should work with it.